Added message when google validations id different from enter validation

Kwasi008 opened the pull request:
- check old address entered to what google validates

micronix commented on Gemfile:
```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```
hey @Kwasi008 why did you changed this to http?

micronix reviewed (changes requested):
Hey @Kwasi008 what is this for? If I understand it correctly. When a user updates a ride and they change the address there is an alert that shows the address changed?

Also can you removed the commented out byebug statements

Kwasi008 commented:
Hi @micronix, I was working on Jira card 26 which was  "As an admin, I want to confirm riders' addresses after validation when creating rides". I can adjust it if that was the desired goal.

kidatsy commented on spec/requests/rides_spec.rb:
```diff
@@ -128,7 +128,6 @@
   it 'will change status to picking up on a ride for a driver' do
     post "/api/v1/rides/#{ride9.id}/picking-up",  headers: {"ACCEPT" => "application/json",  "Token" => "7897"}
     parsed_json = JSON.parse(response.body)
-    byebug
```
So many old byebugs haha!

Kwasi008 commented:
![Screen Shot 2021-01-22 at 2 39 00 PM](https://user-images.githubusercontent.com/34540242/105537412-e2e37b00-5cbf-11eb-991d-c5556c59b43a.png)

Kwasi008 commented:
Hi @micronix 
- changed how the flash message was working, we were not accounting for user alerts, we were just accounting for notice and errors and not alerts.
- Changed the controller's flash.alert into flash.errors when it is an error and not an alert.
- Attach a picture to show the changes

Kwasi008 commented:
@micronix 
- Added clear flash messages when the page is reloading

kidatsy commented on app/helpers/application_helper.rb:
```diff
@@ -7,7 +7,20 @@ def pass_fail(application_state)
+  def bootstrap_class_for(flash_type)
+  case flash_type
+  when "notice"
+      "alert-success"
```
indentation?

Kwasi008 replied:
Thanks TC, I just went through and checked all of them. Please let me know if there anything else I should change on this PR. Thank you

jrmcgarvey reviewed (approved):
Looks OK to me.
